lib/util/package-rules/current-value.ts

```diff
@@ -0,0 +1,31 @@
+import is from '@sindresorhus/is';
+import type { PackageRule, PackageRuleInputConfig } from '../../config/types';
+import { logger } from '../../logger';
+import { configRegexPredicate } from '../regex';
+import { Matcher } from './base';
+
+export class CurrentValueMatcher extends Matcher {
+  override matches(
+    { currentValue }: PackageRuleInputConfig,
+    { matchCurrentValue }: PackageRule
+  ): boolean | null {
+    if (is.undefined(matchCurrentValue)) {
+      return null;
+    }
+    const matchCurrentValuePred = configRegexPredicate(matchCurrentValue);
+
+    if (!matchCurrentValuePred) {
+      logger.warn(
+        { matchCurrentValue },
+        'matchCurrentValue should be a regex, starting and ending with `/`'
+      );
+      return null;
```

Comment:
Should this really be handled as there is no rule at all?     
I think `false` would be better here, if somehow the validation fails the rule should not be applied.

Reply:
Neither case is great, but I agree with you so have switched